statsmodels/sandbox/regression/tests/test_gmm.py

```diff
@@ -237,7 +238,8 @@ def test_hypothesis(self):
         # Smoke test for Wald
         res_wald = res1.wald_test(restriction[:-1])
 
-    def test_smoke(self):
+    @pytest.mark.smoke
+    def test_summary(self):
```

Comment:
there are a few summary test that are mostly smoke tests but have one assert for some details
I'm still fine with marking as smodel